src/ApiDocs.res

```diff
@@ -41,9 +41,58 @@ type item =
       detail: Js.Null.t<detail>,
     })
 
+module RightSidebar = {
+  @react.component
+  let make = (~items: array<item>) => {
+    switch items->Js.Array2.length === 0 {
```

Comment:
redundant length check, you have them as well in both places where you use `<RightSidebar />`

Reply:
I checked because I don't want to show the "Types and Values" text when it is empty.

Reply:
Everywhere where you use the RightSidebar component, you already ensure that the items are not empty. That's why I suggested to remove the one in RightSidebar.

Reply:
Oh right. Removed 5062cd8e